lib/rules/kind-name.js

```diff
@@ -133,14 +127,7 @@ module.exports = {
         (node.parent.type === 'VariableDeclarator' || node.parent.method === true)
       );
 
-      if (
-        namedObjectAssignment || namedObjectDeclaration ||
-        namedClass ||
-        namedFunctionDeclaration || namedFunctionExpression
-      ) {
-        return true;
-      }
-      return false;
+      return !!(namedObjectAssignment || namedObjectDeclaration || namedClass || namedFunctionDeclaration || namedFunctionExpression);
```

Comment:
How about keeping the original code? The logic is the same but for better readability.